src/golang/cmd/server/handler/register_workflow.go

```diff
@@ -260,15 +269,17 @@ func (h *RegisterWorkflowHandler) Perform(ctx context.Context, interfaceArgs int
 		CleanupTimeout:       engine.DefaultCleanupTimeout,
 	}
 
-	_, err = h.Engine.TriggerWorkflow(
-		ctx,
-		workflowId,
-		shared_utils.AppendPrefix(dbWorkflowDag.Metadata.ID.String()),
-		timeConfig,
-		nil, /* parameters */
-	)
-	if err != nil {
-		return emptyResp, http.StatusInternalServerError, errors.Wrap(err, "Unable to trigger workflow.")
+	if args.runNow {
```

Comment:
@saurav-c confirming this change should automatically apply to airflow?

Reply:
Actually this route only registers non-Airflow workflows. Airflow workflows go to the handler defined in `register_airflow_workflow.go`. 

However, since Airflow requires the user to copy over the DAG file, there is no notion of `runNow` for it.